core/src/main/java/org/kiwix/kiwixmobile/core/bookmark/BookmarksActivity.java

```diff
@@ -61,12 +64,16 @@
   private final List<BookmarkItem> allBookmarks = new ArrayList<>();
   private final List<BookmarkItem> deleteList = new ArrayList<>();
 
+  public static final String USER_CLEARED_BOOKMARKS = "user_cleared_history";
```

Comment:
You don't need to do this, the reason we have this field in History is that when you go back to the webview after deleting your history we need to reset it because you would be able to navigate backwards into "deleted" history. The same is not true for bookmarks.

Also the string value being the same as `USER_CLEARED_BOOKMARKS` is confusing, if 2 constants share the same value they should be unified into 1 variable and it renamed to suit its purpose, something like `RESET_WEBVIEW` but this is neither here nor there